app/cmd/snapshot.go

```diff
@@ -469,7 +469,7 @@ func cloneSnapshot(c *cli.Context) error {
 	defer fromControllerClient.Close()
 
 	if err := sync.CloneSnapshot(controllerClient, fromControllerClient, volumeName, fromVolumeName,
-		snapshotName, exportBackingImageIfExist, fileSyncHTTPClientTimeout); err != nil {
+		snapshotName, exportBackingImageIfExist, fileSyncHTTPClientTimeout, 0); err != nil {
```

Comment:
Why not get the arg from `grpc-timeout-seconds` which is also needed to add to the command flag.

Reply:
Thank you. Fixed

Reply:
Btw, for the context, I think the CLI is only used to debugging now since we will use gRPC sorely for communication between longhorn-manager -> instance-manager -> engine -> replica

Reply:
yeah, exactly.